### DISCUSSION TITLE
no le está permitido acceder a esta página o su conexión ha caducado

### DISCUSSION THREAD

**Discussion by edufinci** [chamilo/chamilo-lms#5749] [General]
hola hola 
hace unos dias mi pagina de chamillo no me deja acceder ni a mis alumnos con un mensaje

Lo sentimos, no le está permitido acceder a esta página o su conexión ha caducado. Para volver a la página anterior pulse el enlace inferior o haga clic en el botón "Atrás" de su navegador.

Alguien me puede ayudar un millon de gracias

**Comment by Kaneda-1** [comment]
Hola.  

Con la información que indicas no se puede ayudar mucho ¿cuál es la dirección de tu instalación Chamilo? ¿qué alojamiento tienes?

Un saludo.
